### PR TITLE
Simplify find_package function calls for Qt5 and Qt6

### PR DESCRIPTION
**Pull request by paceholder**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -165,10 +151,10 @@ target_include_directories(QtNodes
...
 target_link_libraries(QtNodes
   PUBLIC
-    ${Qt}::Core
-    ${Qt}::Widgets
-    ${Qt}::Gui
-    ${Qt}::OpenGL
+    Qt${QT_VERSION_MAJOR}::Core
+    Qt${QT_VERSION_MAJOR}::Widgets
+    Qt${QT_VERSION_MAJOR}::Gui
+    Qt${QT_VERSION_MAJOR}::OpenGL
```
Comment (paceholder) [inline CMakeLists.txt:157]: If we require 5.15 as a minimum version, then we'd be able to write just `Qt::Widgets`, such namespaces are defined in 5.15 and 6.xx

File: CMakeLists.txt [hunk 2]
```diff
@@ -37,41 +38,26 @@ if(QT_NODES_DEVELOPER_DEFAULTS)
...
-if (NOT Qt6_FOUND)
-  find_package(Qt5 QUIET
-    COMPONENTS
-     Core
-     Widgets
-     Gui
-     OpenGL
-  )
+if (${QT_VERSION} VERSION_LESS_EQUAL 5.9.0)
+  message(FATAL_ERROR "Requires qt version >= 5.9.0, Your current version is ${QT_VERSION}")
```
Comment (paceholder) [inline CMakeLists.txt:54]: I kept 5.9 for now just not to break the builds of many users.

I still hope to try the fix called "3rd issue fix" from here
https://stackoverflow.com/questions/50064670/qt-library-link-errors-in-windows

@QianMoth  could you maybe have a look at it on windows?

Comment (QianMoth) [reply CMakeLists.txt:54]: os: windows
qt-version: 5.9.0, 5.12.0, 5.15.2, 6.2.1

5.9.0, 5.12.0 : mvsc2017
5.15.2, 6.2.1: mvsc2019

**change :**
```cpp
- Q_NAMESPACE_EXPORT(NODE_EDITOR_PUBLIC)
+ NODE_EDITOR_PUBLIC Q_NAMESPACE
```

**5.12.0, 5.15.2, 6.2.1**
pass

**5.9.0**
```bash
\Software\Qt\5.9\msvc2017_64\lib\Qt5Widgetsd.lib D:\Software\Qt\5.9\msvc2017_64\lib\Qt5Guid.lib D:\Software\Qt\5.9\msvc2017_64\lib\Qt5Cored.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\QtNodes.dir/intermediate.manifest CMakeFiles\QtNodes.dir/manifest.res" failed (exit code 1169) with the following output:
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::AbstractGraphModel::metaObject(void)const " (?metaObject@AbstractGraphModel@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::AbstractGraphModel::qt_metacast(char const *)" (?qt_metacast@AbstractGraphModel@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::AbstractGraphModel::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@AbstractGraphModel@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::AbstractGraphModel::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@AbstractGraphModel@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::connectionCreated(struct QtNodes::ConnectionId)" (?connectionCreated@AbstractGraphModel@QtNodes@@QEAAXUConnectionId@2@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::connectionDeleted(struct QtNodes::ConnectionId)" (?connectionDeleted@AbstractGraphModel@QtNodes@@QEAAXUConnectionId@2@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::nodeCreated(unsigned int)" (?nodeCreated@AbstractGraphModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::nodeDeleted(unsigned int)" (?nodeDeleted@AbstractGraphModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::nodeUpdated(unsigned int)" (?nodeUpdated@AbstractGraphModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::nodeFlagsUpdated(unsigned int)" (?nodeFlagsUpdated@AbstractGraphModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::nodePositionUpdated(unsigned int)" (?nodePositionUpdated@AbstractGraphModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::AbstractGraphModel::modelReset(void)" (?modelReset@AbstractGraphModel@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_AbstractGraphModel.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::AbstractGraphModel::staticMetaObject" (?staticMetaObject@AbstractGraphModel@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::BasicGraphicsScene::metaObject(void)const " (?metaObject@BasicGraphicsScene@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::BasicGraphicsScene::qt_metacast(char const *)" (?qt_metacast@BasicGraphicsScene@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::BasicGraphicsScene::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@BasicGraphicsScene@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::BasicGraphicsScene::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@BasicGraphicsScene@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeMoved(unsigned int,class QPointF const &)" (?nodeMoved@BasicGraphicsScene@QtNodes@@QEAAXIAEBVQPointF@@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeClicked(unsigned int)" (?nodeClicked@BasicGraphicsScene@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeSelected(unsigned int)" (?nodeSelected@BasicGraphicsScene@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeDoubleClicked(unsigned int)" (?nodeDoubleClicked@BasicGraphicsScene@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeHovered(unsigned int,class QPoint)" (?nodeHovered@BasicGraphicsScene@QtNodes@@QEAAXIVQPoint@@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeHoverLeft(unsigned int)" (?nodeHoverLeft@BasicGraphicsScene@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::connectionHovered(struct QtNodes::ConnectionId,class QPoint)" (?connectionHovered@BasicGraphicsScene@QtNodes@@QEAAXUConnectionId@2@VQPoint@@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::connectionHoverLeft(struct QtNodes::ConnectionId)" (?connectionHoverLeft@BasicGraphicsScene@QtNodes@@QEAAXUConnectionId@2@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::BasicGraphicsScene::nodeContextMenu(unsigned int,class QPointF)" (?nodeContextMenu@BasicGraphicsScene@QtNodes@@QEAAXIVQPointF@@@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_BasicGraphicsScene.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::BasicGraphicsScene::staticMetaObject" (?staticMetaObject@BasicGraphicsScene@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_ConnectionGraphicsObject.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::ConnectionGraphicsObject::metaObject(void)const " (?metaObject@ConnectionGraphicsObject@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_ConnectionGraphicsObject.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::ConnectionGraphicsObject::qt_metacast(char const *)" (?qt_metacast@ConnectionGraphicsObject@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_ConnectionGraphicsObject.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::ConnectionGraphicsObject::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@ConnectionGraphicsObject@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_ConnectionGraphicsObject.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::ConnectionGraphicsObject::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@ConnectionGraphicsObject@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_ConnectionGraphicsObject.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::ConnectionGraphicsObject::staticMetaObject" (?staticMetaObject@ConnectionGraphicsObject@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphModel.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::DataFlowGraphModel::staticMetaObject" (?staticMetaObject@DataFlowGraphModel@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphModel.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::DataFlowGraphModel::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@DataFlowGraphModel@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphModel.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::DataFlowGraphModel::metaObject(void)const " (?metaObject@DataFlowGraphModel@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphModel.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::DataFlowGraphModel::qt_metacast(char const *)" (?qt_metacast@DataFlowGraphModel@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphModel.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::DataFlowGraphModel::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@DataFlowGraphModel@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::DataFlowGraphModel::inPortDataWasSet(unsigned int,enum QtNodes::PortType,unsigned int)" (?inPortDataWasSet@DataFlowGraphModel@QtNodes@@QEAAXIW4PortType@2@I@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphicsScene.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::DataFlowGraphicsScene::staticMetaObject" (?staticMetaObject@DataFlowGraphicsScene@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphicsScene.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::DataFlowGraphicsScene::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@DataFlowGraphicsScene@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphicsScene.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::DataFlowGraphicsScene::metaObject(void)const " (?metaObject@DataFlowGraphicsScene@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphicsScene.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::DataFlowGraphicsScene::qt_metacast(char const *)" (?qt_metacast@DataFlowGraphicsScene@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphicsScene.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::DataFlowGraphicsScene::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@DataFlowGraphicsScene@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_DataFlowGraphicsScene.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::DataFlowGraphicsScene::sceneLoaded(void)" (?sceneLoaded@DataFlowGraphicsScene@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_GraphicsView.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::GraphicsView::metaObject(void)const " (?metaObject@GraphicsView@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_GraphicsView.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::GraphicsView::qt_metacast(char const *)" (?qt_metacast@GraphicsView@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_GraphicsView.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::GraphicsView::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@GraphicsView@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_GraphicsView.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::GraphicsView::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@GraphicsView@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_GraphicsView.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::GraphicsView::scaleChanged(double)" (?scaleChanged@GraphicsView@QtNodes@@QEAAXN@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_GraphicsView.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::GraphicsView::staticMetaObject" (?staticMetaObject@GraphicsView@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::NodeDelegateModel::metaObject(void)const " (?metaObject@NodeDelegateModel@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::NodeDelegateModel::qt_metacast(char const *)" (?qt_metacast@NodeDelegateModel@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::NodeDelegateModel::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@NodeDelegateModel@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::NodeDelegateModel::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@NodeDelegateModel@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::dataUpdated(unsigned int)" (?dataUpdated@NodeDelegateModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::dataInvalidated(unsigned int)" (?dataInvalidated@NodeDelegateModel@QtNodes@@QEAAXI@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::computingStarted(void)" (?computingStarted@NodeDelegateModel@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::computingFinished(void)" (?computingFinished@NodeDelegateModel@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::embeddedWidgetSizeUpdated(void)" (?embeddedWidgetSizeUpdated@NodeDelegateModel@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::portsAboutToBeDeleted(enum QtNodes::PortType,unsigned int,unsigned int)" (?portsAboutToBeDeleted@NodeDelegateModel@QtNodes@@QEAAXW4PortType@2@II@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::portsDeleted(void)" (?portsDeleted@NodeDelegateModel@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::portsAboutToBeInserted(enum QtNodes::PortType,unsigned int,unsigned int)" (?portsAboutToBeInserted@NodeDelegateModel@QtNodes@@QEAAXW4PortType@2@II@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: void __cdecl QtNodes::NodeDelegateModel::portsInserted(void)" (?portsInserted@NodeDelegateModel@QtNodes@@QEAAXXZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeDelegateModel.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::NodeDelegateModel::staticMetaObject" (?staticMetaObject@NodeDelegateModel@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeGraphicsObject.cpp.obj : error LNK2005: "public: virtual struct QMetaObject const * __cdecl QtNodes::NodeGraphicsObject::metaObject(void)const " (?metaObject@NodeGraphicsObject@QtNodes@@UEBAPEBUQMetaObject@@XZ) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeGraphicsObject.cpp.obj : error LNK2005: "public: virtual void * __cdecl QtNodes::NodeGraphicsObject::qt_metacast(char const *)" (?qt_metacast@NodeGraphicsObject@QtNodes@@UEAAPEAXPEBD@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeGraphicsObject.cpp.obj : error LNK2005: "public: virtual int __cdecl QtNodes::NodeGraphicsObject::qt_metacall(enum QMetaObject::Call,int,void * *)" (?qt_metacall@NodeGraphicsObject@QtNodes@@UEAAHW4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeGraphicsObject.cpp.obj : error LNK2005: "private: static void __cdecl QtNodes::NodeGraphicsObject::qt_static_metacall(class QObject *,enum QMetaObject::Call,int,void * *)" (?qt_static_metacall@NodeGraphicsObject@QtNodes@@CAXPEAVQObject@@W4Call@QMetaObject@@HPEAPEAX@Z) 已经在 mocs_compilation.cpp.obj 中定义
moc_NodeGraphicsObject.cpp.obj : error LNK2005: "public: static struct QMetaObject const QtNodes::NodeGraphicsObject::staticMetaObject" (?staticMetaObject@NodeGraphicsObject@QtNodes@@2UQMetaObject@@B) 已经在 mocs_compilation.cpp.obj 中定义
  正在创建库 lib\QtNodes.lib 和对象 lib\QtNodes.exp
bin\QtNodes.dll : fatal error LNK1169: 找到一个或多个多重定义的符号
ninja: build stopped: subcommand failed.
```

Comment (QianMoth) [reply CMakeLists.txt:54]: I couldn't find any information about the error reported in version 5.9.0, so I'm uploading it in the hope of getting guidance on how to solve it

**Pull request merged**